### DISCUSSION TITLE
[cloud_functions] (WEB) Workaround to avoid preflight Request.

### DISCUSSION THREAD

**Discussion by kaumudpa** [firebase/flutterfire#5623] [Feature request]
While calling `oncall` http function from Web app there is always a CORS preflight request which ends with status `204` before the actual function starts.

Is there a workaround to avoid this? Since the preflight request invocation is also counted as a invocation towards the allotted quota.

To Reproduce:
1. Call any simple http function via cloud_functions plugin on flutter web.
2. Check the Firebase console, You will find 2 executions of that function - The first one ending with status`204` and the second one ending with `200`. 

The first execution is the preflight request while the second one is the actual execution of our function.

Note : This behaviour is only shown while calling the function from flutter web. On IOS and Andriod the function works completely fine.

**Comment by TahaTesser** [comment]
`cloud_functions: ^0.5.0`

<img width="1128" alt="Screenshot 2020-07-20 at 7 28 12 PM" src="https://user-images.githubusercontent.com/48603081/87946066-2f56bf80-cabf-11ea-83a0-e8b187b4819c.png">

**Comment by noelmansour** [comment]
This seems to be an issue with the js sdk: See https://github.com/firebase/firebase-js-sdk/issues/1800. You can find some workarounds there.
